### PR TITLE
Prevent e.target.classList from being undefined

### PR DESCRIPTION
**Pull request by mselerin**
Sometimes, e.target.classList is undefined for unknow reasons (not tested deeply).

Added an "if" statement to prevent code from crashing.

### CODE REVIEW

File: src/directives/droppable.ts [hunk 1]
```diff
@@ -1,32 +1,3 @@
-import {Directive, ElementRef, HostListener, Input, Output, EventEmitter} from '@angular/core';
```
Comment (ObaidUrRehman) [inline src/directives/droppable.ts]: @mselerin, I think you accidentally deleted these lines?

**Comment by ObaidUrRehman** [comment]
@mselerin I think we need to do the same in draggable directive as well.

File: src/directives/droppable.ts (hunk 1 above)
Comment (mselerin) [inline src/directives/droppable.ts]: Well... This is weird, this line is still in my editor...

OK, I will fix this and btw apply the same fix in the draggable directive.

**Comment by mselerin** [comment]
Fixed in draggable & droppable.

**Comment by ObaidUrRehman** [comment]
Thanks @mselerin !

**Pull request merged**